governance/docs/what_is_governance.md

```diff
@@ -0,0 +1,61 @@
+# What Is Governance?
+
+Because open source projects are organizations, all projects have governance. Some of governance structures are more *explicit* than others. Some are more *formal* than others. But every project has them.
+
+Unfortunately, too many discussions of open source project governance focus on activities or resources, like "speaking for the project" or "ownership of the web domain." While documenting these functions is useful, these are not, strictly speaking, governance issues.  Rather, they are governance outcomes.
+
+At its heart, open source project and community governance is about people: their rights and responsibilities as part of a project, and the expectations others have for them.
+
+## Why governance?
+
+In some open source communities, "governance" gets a bad rap. This is true in cases where project contributors tend to construe governance as a purely negative force ... a set of rules or procedures aimed solely at telling people what they can't do, how they shouldn't act, or how they should limit themselves to acting only within certain boundaries.
```

Comment:
It might be helpful to have examples of where "governance" got a bad rap?

Reply:
How would it be helpful?

Reply:
In framing and dispelling rumors and misconceptions.

Reply:
that would require a substantial addition to the section, in order to bulk out what was just some introductory text. If you have an idea of what you'd like to see here, I'm happy to work on a merge, but to me it seems like a tangent.

Reply:
Closing, due to lack of additional content.